### PR TITLE
document about priorityjob vs reprocessing

### PR DESCRIPTION
**Pull request by peterbe**
@twobraids It's rough and short but did I get the gist right?

### CODE REVIEW

File: docs/development/glossary/index.rst [hunk 1]
```diff
@@ -53,6 +53,11 @@ or Client OS be used instead.
 minidump_stackwalk program are stored. The actual files are stored
 with a .jsonz extension.
 
+**Priority job**: A reprocessing request (see ``Reprocessing``) that has
```
Comment (twobraids) [inline docs/development/glossary/index.rst]: priority jobs only happen to crashes that have never been processed, so it is not a "reprocessing" request.

Comment (peterbe) [inline docs/development/glossary/index.rst]: Yeah. I phrased myself poorly. I tried again.

File: docs/development/glossary/index.rst [hunk 2]
```diff
@@ -53,6 +53,14 @@ or Client OS be used instead.
 minidump_stackwalk program are stored. The actual files are stored
 with a .jsonz extension.
 
+**Priority job**: A processing request (similar ``Reprocessing``) that has
+higher priority. The order of processing across all queues is: Priority,
+Standard, Reprocess, Priority. One crash from each at a time.
+Then it reverses the order and goes through those four again.
+And reverses again. Rinse and repeat. Priority jobs are processing of
+raw crashes that have never been processed. Crashes are put automatically
+on the Priority job queue based on this fact.
```
Comment (twobraids) [inline docs/development/glossary/index.rst]: replace the last sentence with:

Crashes are put on this queue only when a processed crash is requested from the UI and the associated raw crash has not been processed in the past.

Comment (peterbe) [inline docs/development/glossary/index.rst]: Done. Thank you for this.

**Comment by twobraids** [comment]
r+

**Pull request merged**